physics/SFC_Models/Land/Noahmp/module_sf_noahmplsm.F90

```diff
@@ -5738,7 +5777,7 @@ subroutine gfs_stability                                              &
           tem1      = 0.00001_kind_phys/z1                                       ! minimum exhange coef (?)
           cm        = max(cm, tem1)
           ch        = max(ch, tem1)
-          stress    = cm * wind * wind                                    ! surface stress = Cm*U*U
+          stress    = (cm + cf)* wind * wind  ! cf for tofd  10.15) or (10.19) in Arya
```

Comment:
Add the Arya reference above where UTN and STN are defined. If you are referencing Arya's book, mine doesn't have these equations. the cf reference should be used in 5764 above instead of this line

Reply:
comment removed